app/uk/gov/hmrc/eusubsidycompliance/connectors/EisConnector.scala

```diff
@@ -138,38 +125,31 @@ class EisConnector @Inject()(
     )(implicitly, implicitly, addHeaders, implicitly)
   }
 
-  def updateSubsidy(
-    subsidyUpdate: SubsidyUpdate
-  )(
-    implicit hc: HeaderCarrier,
-    ec: ExecutionContext
-  ): Future[Unit] = {
-
-    import uk.gov.hmrc.eusubsidycompliance.models.json.digital.amendSubsidyResponseReads
+  def updateSubsidy(subsidyUpdate: SubsidyUpdate)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] = {
 
     val eisTokenKey = "eis.token.scp06"
+
     desPost[SubsidyUpdate, Unit](
       s"$eisURL/$amendSubsidyPath",
       subsidyUpdate, eisTokenKey
     )(implicitly, implicitly, addHeaders, implicitly)
   }
 
-  // TODO - modify this to accept date range.
   def retrieveSubsidies(
-                       ref: UndertakingRef
-                       )(
-    implicit hc: HeaderCarrier,
-    ec: ExecutionContext
-                       ): Future[UndertakingSubsidies] = {
-    val eisTokenKey = "eis.token.scp09"
+    ref: UndertakingRef,
+    dateRange: Option[(LocalDate, LocalDate)]
+  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[UndertakingSubsidies] = {
+
     import uk.gov.hmrc.eusubsidycompliance.models.json.eis.eisRetrieveUndertakingSubsidiesResponseWrite
 
+    val eisTokenKey = "eis.token.scp09"
+
+    val defaultDateRange = Some((LocalDate.of(2000, 1, 1), LocalDate.now()))
```

Comment:
Should we store this date in a val  in case we decide to change the range in future?